README.md

```diff
@@ -65,3 +74,7 @@ Contributions are welcome! Please read [CONTRIBUTING.md](CONTRIBUTING.md) for de
 ## License
 
 This project is licensed under the [MIT License](LICENSE).
+
+### Discord
+
+Join Our [Discord Community](https://discord.gg/PYFA9Hwp) for any help and troubleshooting.
```

Comment:
On version2, we'll have link as /discord (so keep it as `https://digitomize.com/discord`).
Like: https://v2.digitomize.com/discord